### PR TITLE
Add sourcing the virtual environment to the right spot

### PR DESCRIPTION
**Pull request by LeStarch**
| | |
|:---|:---|
|**_Originating Project/Creator_**| |
|**_Affected Component_**|  |
|**_Affected Architectures(s)_**|  |
|**_Related Issue(s)_**|  |
|**_Has Unit Tests (y/n)_**|  |
|**_Builds Without Errors (y/n)_**|  |
|**_Unit Tests Pass (y/n)_**|  |
|**_Documentation Included (y/n)_**|  |

---
## Change Description

Adjusting the hello-world tutorial with feedback.

### CODE REVIEW

**Comment by LeStarch** [comment]
@thomas-bc and @jwest115, I did another pass at the hello world tutorial. Here are some minor edits and QoL improvements.

File: docs/Tutorials/HelloWorld/HelloWorld.md [hunk 1]
```diff
@@ -104,21 +107,23 @@ To build this component run `fprime-util build` in the current folder.
...
+async command TODO opcode 0
+```
 
 with the following:
 
 ```
 @ Command to issue greeting with maximum length of 20 characters
 async command SAY_HELLO(
-    greeting: string size 20 @ Greeting to repeat in the Hello event
+    greeting: string size 20 @< Greeting to repeat in the Hello event
```
Comment (thomas-bc) [inline docs/Tutorials/HelloWorld/HelloWorld.md:121]: This was already updated in `devel` in https://github.com/nasa/fprime/pull/1985/files so I'm confused as to why it still shows up here (I understand you probably branched off earlier than the change). Could that cause a merge conflict?

**Review by thomas-bc — APPROVED** [review]
LGTM!

File: docs/Tutorials/HelloWorld/HelloWorld.md (hunk 1 above)
Comment (thomas-bc) [reply docs/Tutorials/HelloWorld/HelloWorld.md:121]: Apparently no conflict.

**Review by jwest115 — APPROVED** [review]
looks good!

**Pull request merged**